test/test_core.py

```diff
@@ -9359,20 +9334,12 @@ def test_Module_dynamicLibraries(self, args):
       force_c=True)
 
   # Tests the emscripten_get_exported_function() API.
+  @also_with_minimal_runtime
   def test_get_exported_function(self):
     self.set_setting('ALLOW_TABLE_GROWTH')
     self.emcc_args += ['-lexports.js']
     self.do_core_test('test_get_exported_function.cpp')
 
-  # Tests the emscripten_get_exported_function() API.
-  @no_asan('TODO: ASan support in minimal runtime')
```

Comment:
Does `@also_with_minimal_runtime` somehow skip asan automatically?

Reply:
No, but it maybe it should?   Let me check

Reply:
The test now passes under Asan and minimal runtime!

Reply:
I split that out into its own change: https://github.com/emscripten-core/emscripten/pull/23562

Reply:
OK, #23562 has now landed so there should be no more change here WRT asan.